nbdev/doclinks.py

```diff
@@ -127,23 +127,28 @@ def nbglob_cli(
                   skip_file_glob=skip_file_glob, skip_file_re=skip_file_re, skip_folder_re=skip_folder_re)
 
 # %% ../nbs/api/05_doclinks.ipynb 22
+# the import is needed to call getattr on it
+import nbdev.export
+
 @call_parse
 @delegates(nbglob_cli)
 def nbdev_export(
     path:str=None, # Path or filename
+    procs:str="black_format", # whitespace delimited tokens of processors to use.
```

Comment:
The right way to do it IMO is to use a tuple. That's the normal approach used in most Python APIs.

Reply:
👍 good call. Thank you. I'll look into passing in a tuple via commandline with call_parse. Could probably leverage "choices". I need to see how to set that up, so it might not make it into this PR.

Reply:
I couldn't quite figure out how to get it to work with a tuple.

But using [`Param(.., nargs="*", choices=optional_procs())="black_format"`](https://docs.python.org/3/library/argparse.html#nargs) seems to get something more user-friendly for the command line.

<details>
<summary>Here are some example using `raise Exception(L(procs))` in the method to debug it.</summary>

```
> nbdev_export -h
usage: nbdev_export [-h] [--path PATH] [--symlinks] [--file_glob FILE_GLOB] [--file_re FILE_RE] [--folder_re FOLDER_RE] [--skip_file_glob SKIP_FILE_GLOB] [--skip_file_re SKIP_FILE_RE] [--skip_folder_re SKIP_FOLDER_RE] [--procs [{black_format,scrub_magics} [{black_format,scrub_magics} ...]]]
...
  --procs [{black_format,scrub_magics} [{black_format,scrub_magics} ...]]  tokens naming the export processors to use. (default: black_format)

# default behavior is to pass in black_format
> nbdev_export 
Exception: ['black_format']

# but this default can be omitted
> nbdev_export --procs 
> Exception: []

> nbdev_export --procs scrub_magics black_format
Exception: ['scrub_magics', 'black_format']

> nbdev_export --procs black_format
Exception: ['black_format']

> nbdev_export --procs scrub_magics
Exception: ['scrub_magics']

> nbdev_export --procs black_forma
usage: nbdev_export [-h] [--path PATH] [--symlinks] [--file_glob FILE_GLOB] [--file_re FILE_RE] [--folder_re FOLDER_RE] [--skip_file_glob SKIP_FILE_GLOB] [--skip_file_re SKIP_FILE_RE] [--skip_folder_re SKIP_FOLDER_RE] [--procs [{black_format,scrub_magics} [{black_format,scrub_magics} ...]]]
nbdev_export: error: argument --procs: invalid choice: 'black_forma' (choose from 'black_format', 'scrub_magics')

> nbdev_export --procs black_forma scrub_magics
usage: nbdev_export [-h] [--path PATH] [--symlinks] [--file_glob FILE_GLOB] [--file_re FILE_RE] [--folder_re FOLDER_RE] [--skip_file_glob SKIP_FILE_GLOB] [--skip_file_re SKIP_FILE_RE] [--skip_folder_re SKIP_FOLDER_RE] [--procs [{black_format,scrub_magics} [{black_format,scrub_magics} ...]]]
nbdev_export: error: argument --procs: invalid choice: 'black_forma' (choose from 'black_format', 'scrub_magics')

> nbdev_export --procs scrub_magics barst
usage: nbdev_export [-h] [--path PATH] [--symlinks] [--file_glob FILE_GLOB] [--file_re FILE_RE] [--folder_re FOLDER_RE] [--skip_file_glob SKIP_FILE_GLOB] [--skip_file_re SKIP_FILE_RE] [--skip_folder_re SKIP_FOLDER_RE] [--procs [{black_format,scrub_magics} [{black_format,scrub_magics} ...]]]
nbdev_export: error: argument --procs: invalid choice: 'barst' (choose from 'black_format', 'scrub_magics')
```

</details>